doc/help/kdb-restore.md

```diff
@@ -17,7 +17,39 @@ This command writes into the `/etc` directory and as such it requires root permi
 ## EXAMPLES
 
 ```
```

Comment:
As far as I know, you need to start the code block with:
_```sh_ 
if you want that the examples are verified by the shell recorder.
```suggestion
```sh
```

Reply:
Exactly! tests/shell/shell_recorder/tutorial_wrapper/README.md

Reply:
Still open.